locales/templates/app.pot

```diff
@@ -1965,11 +1964,11 @@ msgstr ""
 
 #. "I'm In!" is a button where users can press to join the reward program
 msgctxt "msrp_challenge_button_1"
-msgid "I’m In!"
+msgid "I'm In!"
 msgstr ""
 
 msgctxt "msrp_challenge_tou"
-msgid "By clicking “I’m in” you are agreeing to our Terms of Use"
+msgid "By clicking “Im In!” you are agreeing to our Terms of Use"
```

Comment:
apostrophe was dropped?

Reply:
The string we have was "By clicking “I\’m In!” you are agreeing to our Terms of Use", is it because we added "\" that it became Im?